packages/strapi-admin/admin/src/components/LeftMenuLinkHeader/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, createRef, useEffect } from 'react';
+import { camelCase } from 'lodash';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import messages from '../LeftMenuLinkContainer/messages.json';
+import Search from './Search';
+
+const Title = styled.div`
```

Comment:
Can you put these components in a separate file? Also can you add `defaultProps` to it in preparation for the theming (which I saw you already started to implement) but let say someone provides a custom theme in his app which doesn't have the `leftMenu` key the component will break. It can be avoided by using `defaultProps`

Reply:
I think that default theme values should not be in a default props object. We should create a default theme that could be overridden with a custom theme. If we put default props to define default values of the theme here, we will have to define theme default props every time we use it (so, in every styled-components).

Reply:
Sure I do agree! However, since this is not implemented yet, I believe that we should use the default props for now like specified in the [docs](https://styled-components.com/docs/advanced#theming)